examples/run_field_dep_justMw.py

```diff
@@ -153,7 +152,7 @@
 sweep_data.set_units("t2", "s")
 
 sweep_data.name(config_dict["type"] + "_" + str(config_dict["field_counter"]))
-sweep_data.set_prop("postproc_type", "field_sweep_v3")
+sweep_data.set_prop("postproc_type", "spincore_generalproc_v1")
```

Comment:
Again, you need to choose a different post-processing type.

Reply:
It seems like it should just be Field sweep version 4

Reply:
I compared to the load_data in the master branch and there is not a field_sweep_v3 yet so this change is actually not necessary!